catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/FederationAdminServiceImpl.java

```diff
@@ -411,6 +423,23 @@ public void deleteRegistryEntriesByMetacardIds(List<String> metacardIds,
         return registryEntries;
     }
 
+    @Override
+    public void refreshRegistriesForSubscriptions() throws FederationAdminException {
+        LOGGER.info("=-=-=-=-=-= REFRESH REGISTRIES {} =-=-=-=-=-=", java.time.LocalTime.now());
```

Comment:
Ha ha. Yeah. That will get pulled out when this is finalized. Just using it as a quick visual reference for when the timer is triggering.